reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/Timeout.kt

```diff
@@ -0,0 +1,42 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.base.CompositeDisposableObserver
+import com.badoo.reaktive.base.exceptions.TimeoutException
+import com.badoo.reaktive.completable.CompletableCallbacks
+import com.badoo.reaktive.disposable.Disposable
+import com.badoo.reaktive.disposable.addTo
+import com.badoo.reaktive.scheduler.Scheduler
+
+fun <T> Observable<T>.timeout(timeoutMillis: Long, scheduler: Scheduler, other: Observable<T>? = null): Observable<T> =
```

Comment:
`scheduler: Scheduler = computationScheduler`?

Reply:
At the moment we do not define default values for scheduler arguments. E.g. debounce  or sample. I find it error prone, since you may expect emissions to be on main thread.